src/sentry/static/sentry/app/components/group/suggestedOwnerHovercard.jsx

```diff
@@ -179,6 +215,21 @@ const OwnershipTag = styled(({tagType, ...props}) => <div {...props}>{tagType}</
   text-align: center;
 `;
 
+const ViewMoreButton = styled(p => (
+  <button className="btn btn-link" {...p}>
```

Comment:
Can we use the `<Button>` component here? We're phasing out the use of `btn` class names (unless it's used in an old button group). `<Button>` is pretty opinionated for padding, but I think we may be able to use `priority=link` w/ `size="zero"` - but if it's a pain, then we can use `Link`.

Reply:
Works well